From 8f13f61a0340e325db02c46bd66e8de9204d5866 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Mon, 20 Aug 2018 18:29:32 +0300
Subject: [PATCH] Makefile: change jenkins project name

The jenkins projects do not like spaces inside the name.
Fix this.

Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/Makefile b/Makefile
index d71277f..e90303d 100644
--- a/Makefile
+++ b/Makefile
@@ -75,8 +75,8 @@
 	@-repo forall $(RELEASE_REPOS) -c git push origin :$(RELEASE_TAG_POINTER)
 	@repo forall $(RELEASE_REPOS) -c git push origin $(HL_FULL_VER) $(RELEASE_TAG_POINTER)
 	@echo "Released: $(HL_FULL_VER)"
-	@wget http://192.168.0.4:8080/job/goya%20release%20test/build?token=goya-release
-	@echo "Started the \"goya release test\" jenkins job... check your email for results."
+	@wget http://192.168.0.4:8080/job/goya-release-test/build?token=goya-release
+	@echo "Started the \"goya-release-test\" jenkins job... check your email for results."
 
 .PHONY: hl-dali hl-gaudi
 hl-dali hl-gaudi: $(OUTPUT_DIR)
